src/store/modules/facility/actions.ts

```diff
@@ -183,6 +183,7 @@ const actions: ActionTree<FacilityState, RootState> = {
     const current = cachedFacilities.find((facility: any) => facility.facilityId === payload.facilityId)
     if(current?.facilityId && !payload.skipState) {
       commit(types.FACILITY_CURRENT_UPDATED, current);
+      await dispatch('fetchFacilityAdditionalInformation', payload);
```

Comment:
As we have found the current facility in list, its additional information is also available so we should not fetch the complete information again just to get inventory Groups.